_maps/RandomRooms/greytopia.dmm

```diff
@@ -0,0 +1,562 @@
+	w_class = 3
+	},
+/obj/item/melee/baton/cattleprod{
+	name = "compact stunprod";
+	w_class = 3
+	},
+/obj/item/stock_parts/cell/high{
+	pixel_x = 6;
+	pixel_y = -3
+	},
+/obj/item/stock_parts/cell/high,
+/obj/item/restraints/legcuffs/bola/tactical,
+/obj/item/restraints/legcuffs/bola/tactical,
+/turf/open/floor/plating,
+/area/greytopia)
+"E" = (
+/obj/structure/chair{
+	dir = 1
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"F" = (
+/obj/structure/table,
+/obj/item/reagent_containers/food/snacks/salad/validsalad{
+	pixel_x = 5;
+	pixel_y = 10
+	},
+/obj/item/reagent_containers/food/snacks/salad/validsalad{
+	pixel_y = -2
+	},
+/obj/item/reagent_containers/food/snacks/salad/validsalad{
+	pixel_x = -2;
+	pixel_x = -2;
+	pixel_y = 4
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"G" = (
+/obj/machinery/limbgrower,
+/turf/open/floor/plating,
+/area/greytopia)
+"H" = (
+/obj/structure/rack,
+/obj/item/twohanded/spear{
+	pixel_x = 7
+	},
+/obj/item/twohanded/spear{
+	pixel_x = -5
+	},
+/obj/item/twohanded/spear{
+	pixel_x = 2
+	},
+/obj/item/twohanded/spear/grey_tide{
+	pixel_x = -2
+	},
+/obj/item/twohanded/spear{
+	pixel_x = 2
+	},
+/obj/item/restraints/handcuffs/cable,
+/obj/item/restraints/handcuffs/cable,
+/obj/item/reagent_containers/spray,
+/obj/item/reagent_containers/spray,
+/turf/open/floor/plating,
+/area/greytopia)
+"I" = (
+/turf/open/floor/plating{
+	icon_state = "panelscorched"
+	},
+/area/greytopia)
+"J" = (
+/obj/structure/bonfire/dense,
+/turf/open/floor/plating,
+/area/greytopia)
+"K" = (
+/obj/item/storage/toolbox/mechanical/old,
+/turf/open/floor/plating,
+/area/greytopia)
+"L" = (
+/obj/structure/table/reinforced,
+/obj/item/reagent_containers/glass/beaker/synthflesh{
+	pixel_x = -5
+	},
+/obj/item/reagent_containers/glass/beaker/synthflesh{
+	pixel_x = 2
+	},
+/obj/item/reagent_containers/spray/medical/sterilizer{
+	pixel_x = 8
+	},
+/obj/item/candle/infinite{
+	pixel_x = -3;
+	pixel_y = 9
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"M" = (
+/obj/structure/rack,
+/obj/item/screwdriver/caravan,
+/obj/item/crowbar/red/caravan,
+/obj/item/wirecutters/caravan{
+	pixel_y = 6
+	},
+/obj/item/wrench/caravan,
+/obj/item/weldingtool/experimental,
+/obj/item/clothing/gloves/color/yellow,
+/turf/open/floor/plating,
+/area/greytopia)
+"N" = (
+/obj/item/banner/greytide{
+	icon_state = "banner_grey"
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"O" = (
+/obj/machinery/shower{
+	dir = 1
+	},
+/turf/open/floor/plating{
+	icon_state = "platingdmg1"
+	},
+/area/greytopia)
+"P" = (
+/obj/structure/table/reinforced,
+/obj/item/soap,
+/obj/item/surgical_drapes{
+	pixel_z = 8
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"Q" = (
+/obj/structure/fans/tiny,
+/obj/machinery/door/airlock/vault/scanner,
+/turf/open/floor/plating,
+/area/greytopia)
+"R" = (
+/obj/machinery/doorscanner,
+/turf/open/floor/plating,
+/area/greytopia)
+"S" = (
+/obj/structure/falsewall{
+	layer = 3.3
```

Comment:
layer?

Reply:
it was because the turf only had a poster as a "hint" to its presence but false walls eat posters